inspec.yml

```diff
@@ -23,4 +23,22 @@ inputs:
     desc: "This details an organization-approved encrypted search string to perform encrypted searches"
     type: String
     value: ''
+
+  - name: google_chrome_version
+    desc: "This may be manually entered and be found by opening the Chrome browser, going to Settings, then clicking About Chrome (at the bottom) "
+    type: String
+    value: ''    
+
+  - name: siprnet_system
+    description: "If this system/machine is on the SIPRNet"
+    type: Boolean
+    value: false
+
+ - name: administrator_approved_urls
```

Comment:
There is a missing space before `- name: administrator_approved_urls`, which leads to error `did not find expected key while parsing a block mapping`. Consider adding an additional space before `- name: administrator_approved_urls` to fix the syntax.